content/markdown/02-working-with-data/02-06-factors.rmd

```diff
@@ -0,0 +1,189 @@
+---
+output:
+  pdf_document: default
+  html_document: default
+---
+# Factors - Releveling & Recoding
```

Comment:
I think it would be worth introducing the different types of data early. Talking about re-leveling and recoding factors is fine here, but it would be better to introduce what a factor is earlier. When they learn to import data might be a good part. They can look at the structure of the data they imported using str(data) and describe the different types of data in each column